packages/docs/stories/templates/form.stories.tsx

```diff
@@ -178,7 +178,12 @@ export const ContactForm = {
               `)}
             </syn-select>
 
-            <syn-input label="${getTranslation('contactForm.referenceContactLabel')}" name="salesPerson" required></syn-input>
+            <syn-combobox label="${getTranslation('contactForm.referenceContactLabel')}" name="salesPerson" required>
```

Comment:
The flyout has some weird graphic behaviour, propably hoisting?

Reply:
Seems like this is not only a problem of the combobox, but also with the select.
On the left is the combobox, on the right the select: 


![image](https://github.com/user-attachments/assets/85cab764-0540-4a26-8c2c-28ad1f7ba2db)

Reply:
this can't be fixed via hoist :/

Reply:
I created an issue in [our backlog](https://github.com/synergy-design-system/synergy-design-system/issues/612) and in [shoelace](https://github.com/shoelace-style/shoelace/issues/2171).

I added a temporary fix in the form template. As soon as the issue is fixed ( if even possible) this can be removed.